aws/cloudformation/lambdas/sql_user/sql_user.rb

```diff
@@ -0,0 +1,98 @@
+require 'mysql2'
+require 'json'
+require_relative './cfn_response'
```

Comment:
Is this meant to refer to https://github.com/code-dot-org/code-dot-org/blob/staging/aws/cloudformation/cfn_response.rb? If so, it should probably be more like 
```suggestion
require_relative '../../cfn_response'
```